sysutils/mise/Portfile

```
@@ -1,14 +1,14 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
-PortGroup           cargo 1.0
+PortGroup           github  1.0
+PortGroup           cargo   1.0
 
-github.setup        jdx mise 2024.6.6 v
+github.setup        jdx mise 2024.8.3 v
```

```suggestion
github.setup        jdx mise 2024.8.4 v
```

2024.8.4 was just released. It doesn't appear to change anything from our end except dependencies.